### PR TITLE
check/update/fix tutorials

### PR DESCRIPTION
**Pull request by sduquemesa**
**Context:** The `teleportation.py` tutorial in the examples folder is very outdated, and uses functionality latest seen in v0.11.0 (e.g. the `strawberryfields.utils.scale` function). It should either be updated or removed.

**Description of the Change:** Check, update and fix strawberryfields tutorials in the examples folder 

**Benefits:** Updated tutorials and examples

**Possible Drawbacks:** None

**Related GitHub Issues:** [Update `examples/teleportation.py` (#612)](https://github.com/XanaduAI/strawberryfields/issues/612)

### CODE REVIEW

File: examples/teleportation.py [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
+    MeasureX | q[0]
+    MeasureP | q[1]
+
+    # Displacement gates conditioned on
+    # the measurements
+    Xgate(sqrt(2) * q[0].par) | q[2]
+    Zgate(sqrt(2) * q[1].par) | q[2]
+
+eng = sf.Engine("fock", backend_options={"cutoff_dim": 15})
+result = eng.run(prog, shots=1, modes=None, compile_options={})
```
Comment (thisac) [inline examples/teleportation.py:46]: (minor) maybe just leave these as default (which coincidentally is exactly the explicit ones here).

```suggestion
result = eng.run(prog)
```

File: examples/teleportation.py [hunk 2]
```diff
@@ -1,41 +1,51 @@
 #!/usr/bin/env python3
 import strawberryfields as sf
-from strawberryfields.ops import *
-from strawberryfields.utils import scale
```
Comment (thisac) [inline examples/teleportation.py:4]: Was `scale` actually never used? 🤔

**Review by thisac — APPROVED** [review]
Looks good! 👍

File: examples/teleportation.py (hunk 2 above)
Comment (sduquemesa) [reply examples/teleportation.py:4]: nope, it wasn't used 🙃

**Pull request merged**